database/migrations/2021_09_06_074645_update_users_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->bigIncrements('id');
```

Comment:
xem lại toàn bộ cho anh bảng users này, đây là update hoặc alter migration nhé, không create như thế này

Reply:
@hapo-nghialuu Em tìm hiểu tài liệu thì thấy Schema::create là tạo bảng, Schema::table là update bảng ạ